express/blocks/columns/columns.js

```diff
@@ -132,6 +205,10 @@ export default function decorate($block) {
       // this probably needs to be tighter and possibly earlier
       const $a = $cell.querySelector('a');
       if ($a) {
+        if ($a.href.includes('vimeo') && $row.parentElement.classList.contains('highlight')) {
```

Comment:
`displayVideo()` can handle YT videos, too, so let's add it here pre-emptively.
```suggestion
        if (($a.href.includes('vimeo') || $a.href.includes('youtu'))
          && $row.parentElement.classList.contains('highlight')) {
```